packages/cli/src/telemetry/telemetry-event-relay.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Service } from 'typedi';
+import { EventRelay } from '@/eventbus/event-relay.service';
+import type { Event } from '@/eventbus/event.types';
+import { Telemetry } from '.';
+import config from '@/config';
+
+@Service()
+export class TelemetryEventRelay {
```

Comment:
do we need this service? can this code not all be in `Telemetry` ?

Reply:
My reasoning is that once finished this will be a large class and I find there's a clear distinction between relaying events and actual telemetry logic - we're unlikely to be interested in both when fixing or debugging something. Same reasoning for not merging `AuditEventRelay` into `MessageEventBus`.